workflow/models.py

```diff
@@ -863,6 +864,8 @@ def get_awaiting_approval(self):
 
     def get_rejected(self):
         return self.filter(approval="rejected")
+    def get_new(self):
+        return self.filter(Q(Q(approval=None) | Q(approval="")))
```

Comment:
The outer "Q" object is unnecessary. You could remove it and make it look like: 
`return self.filter(Q(approval=None) | Q(approval=""))`